src/app/view/components/dot-large-message-display/dot-large-message-display.component.ts

```diff
@@ -2,43 +2,51 @@ import {
     DotLargeMessageDisplayService,
     DotLargeMessageDisplayParams
 } from './services/dot-large-message-display.service';
-import { Component, OnInit, ViewChild, ElementRef, Renderer2, OnDestroy } from '@angular/core';
+import { Component, OnInit,  Renderer2, OnDestroy, ViewChildren, QueryList, AfterViewInit } from '@angular/core';
 import { Subject } from 'rxjs';
 import { takeUntil } from 'rxjs/operators';
+import { DotDialogComponent } from '@components/dot-dialog/dot-dialog.component';
 
 @Component({
     selector: 'dot-large-message-display',
     templateUrl: './dot-large-message-display.component.html',
     styleUrls: ['./dot-large-message-display.component.scss']
 })
-export class DotLargeMessageDisplayComponent implements OnInit, OnDestroy {
-    @ViewChild('body') bodyEl: ElementRef;
-
-    data: DotLargeMessageDisplayParams;
+export class DotLargeMessageDisplayComponent implements OnInit, OnDestroy, AfterViewInit {
+    @ViewChildren(DotDialogComponent) dialogs: QueryList<DotDialogComponent>;
 
+    messages: DotLargeMessageDisplayParams[] = [];
     private destroy$: Subject<boolean> = new Subject<boolean>();
+    private recentlyDialogAdded: boolean;
 
     constructor(
         private dotLargeMessageDisplayService: DotLargeMessageDisplayService,
         private renderer: Renderer2
     ) {}
 
+    ngAfterViewInit() {
+        this.dialogs.changes.subscribe((dialogs: QueryList<DotDialogComponent>) => {
```

Comment:
We're not killing this sub but more than that I think we can use async pipe for messages here.